### PR TITLE
Added Mess tray item

### PR DESCRIPTION
**Pull request by RarkGrames**
#### Summary
```SUMMARY: Content "Added mess tray item"```

#### Purpose of change
Breaking down #37024 to smaller portions for easier review.

#### Describe the solution
- Adding the item
- Adding it to the bunker mapgen file

#### Testing
Spawned in the bunker and checked if the item spawns.

### CODE REVIEW

**Comment by I-am-Erk** [comment]
I'd suggest calling this a "food tray", so it's more generic for use elsewhere

**Comment by RarkGrames** [comment]
I was thinking of [this](https://www.amazon.com/Winco-SMT-1-Mess-Tray-Set/dp/B00OACLMYW). It has containing like the plates in game.

It was easier to put this tray and a fork per person in the canteen and call it a day, rather than to be messing around with simple trays, plates, etc. I can replace them with plastic plates already in the game if that's better.

File: data/json/items/generic/dining_kitchen.json [hunk 1]
```diff
@@ -841,5 +841,22 @@
     "volume": "900 ml",
     "bashing": 7,
     "price": 200
+  },
+  {
+    "id": "mess_tray",
+    "name": "mess tray",
```
Comment (I-am-Erk) [inline data/json/items/generic/dining_kitchen.json]:
```suggestion
    "name": "meal tray",
```

File: data/json/items/generic/dining_kitchen.json [hunk 2]
```diff
@@ -841,5 +841,22 @@
...
+    "description": "A stainless steel tray used for serving food in mess halls or similar places.",
```
Comment (I-am-Erk) [inline data/json/items/generic/dining_kitchen.json]:
```suggestion
    "description": "A stainless steel tray used for serving food in cafeterias, mess halls, or similar places.",
```

**Pull request merged**